src/main/java/seedu/taassist/model/session/Session.java

```diff
@@ -68,6 +90,6 @@ public int hashCode() {
      * Formats state as text for viewing.
      */
     public String toString() {
-        return sessionName;
+        return String.format("%s; Date: %s", getSessionName(), getDate());
```

Comment:
Do we really want to show the user the date of the session and include it as part of the `toString`, since the original intention was for it to be used for ordering the sessions in a list?